numpy/lib/function_base.py

```diff
@@ -3,6 +3,8 @@
 import re
 import sys
 import warnings
+from enum import Enum
+from typing import Tuple
```

Comment:
```suggestion
```
If you're interested: annotations (of the public API) should go in the [function_base.pyi](https://github.com/numpy/numpy/blob/main/numpy/lib/function_base.pyi) stub file.